lib/ws-channel.js

```diff
@@ -0,0 +1,250 @@
+module.exports = WebsocketChannel;
+
+var assert = require('assert');
+var crypto = require('crypto');
+var debug = require('./debug')('ws-channel');
+var events = require('events');
+var extend = require('util')._extend;
+var util = require('util');
+var Websocket = require('ws');
+
+var REQ = 'strong-control-channel:request';
+var RSP = 'strong-control-channel:response';
+var NOT = 'strong-control-channel:notification';
+
+
+function WebsocketChannel(onRequest, onNotification) {
```

Comment:
But FYI, the original addition of notifications had them seperate from requests on the receiver side, and I probably didn't unify all the places.